docs/en/sql-reference/functions/encoding-functions.md

```diff
@@ -206,6 +206,141 @@ Result:
 └──────┘
 ```
 
+## bin {#bin}
+
+Returns a string containing the argument’s binary representation.
+
+Alias: `BIN`.
+
+**Syntax**
+
+``` sql
+bin(arg)
+```
+
+For integer arguments, it prints bin digits (“nibbles”) from the most significant to least significant (big endian or “human readable” order). It starts with the most significant non-zero byte (leading zero bytes are omitted) but always prints eight digits of every byte  if leading digit is zero.
```

Comment:
```suggestion
For integer arguments, it prints bin digits (“nibbles”) from the most significant to least significant (big-endian or “human-readable” order). It starts with the most significant non-zero byte (leading zero bytes are omitted) but always prints eight digits of every byte if a leading digit is zero.
```

Reply:
Nibble is not a binary digit. (It is half-byte).